app/code/community/Oyst/OneClick/Block/Adminhtml/System/Config/Form/Field/Config.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * This file is part of Oyst_OneClick for Magento.
+ *
+ * @license http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
+ * @author Oyst <plugin@oyst.com> <@oyst>
+ * @category Oyst
+ * @package Oyst_OneClick
+ * @copyright Copyright (c) 2017 Oyst (http://www.oyst.com)
+ */
+
+/**
+ * Adminhtml System Config Form Field Config Block
+ */
+class Oyst_OneClick_Block_Adminhtml_System_Config_Form_Field_Config extends Mage_Adminhtml_Block_System_Config_Form_Field
+{
+    protected function _getElementHtml(Varien_Data_Form_Element_Abstract $element)
+    {
+        $trad = array(
+            'postShimpentsSuccess' => $this->__('Delivery mode synchronization was successful.'),
+            'postShimpentsError' => $this->__('The synchronization of delivery modes failed.'),
+            'saveBeforePost' => $this->__('Save the configuration before sending.'),
+            'jsonBadFormat' => $this->__('Bad Json formatting. Use JSON Validator to fix it.'),
+        );
+
+        $shipmentsConfig = Mage::getStoreConfig("oyst/oneclick/shipments_config");
+
+        $output = '<style type="text/css">
+                #oyst_oneclick_oneclick_shipments_config { font-family: courier, courier new, serif; height: 30em; }
+            </style>
+            <script type="text/javascript">
+            //<![CDATA[
+                function showMessage(txt, type) {
+                    var html = \'<ul class="messages"><li class="\' + type + \'-msg"><ul><li>\' + txt + \'</li></ul></li></ul>\';
+                    $("messages").update(html);
+                    var url = location.href;
+                    var n = url.indexOf("#");
+                    url = url.substring(0, n != -1 ? n : url.length);
```

Comment:
`!==`

Reply:
I changed theses three lines for @jney suggestion.